.github/schema-store-issue-template.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "chore: add {{ env.TAG }} to schema store"
+---
+JSON schema released: {{ env.TAG }}. 
+This needs to be added to the schema store catalogue.
```

Comment:
Can you add a link to the scheme store repo? Perhaps also include an example explaining what needs to be changed.

Reply:
done, included example in initial comment